File: 프로그래머스/2/12980. 점프와 순간 이동/점프와 순간 이동.js
function solution(n)
{
    var ans = 0;
    var cnt = 0;
    while (n != 1){
        if (n%2 == 0) { //짝수이면
            n = n/2 
        }
        else {
            n--
            cnt++
        }
    }
    ans = cnt+1
    
    return ans;
}

File: 프로그래머스/2/12980. 점프와 순간 이동/점프와 순간 이동.java
import java.util.*;

public class Solution {
    public int solution(int n) {
        int ans = 0;
        // 2로 안 나뉘어 떨어지면 ans++;
        // -1 하고, 다시 2로 나누기
        while (n != 0) {
            if (n % 2 == 0) {
                n /= 2;
                continue;
            }
            n--;
            ans++;
        }
        

        return ans;
    }
}